rest/blip_sync.go

```diff
@@ -102,17 +102,19 @@ func (sc *ServerContext) incrementConcurrentReplications(ctx context.Context) (b
 		return false, base.ErrReplicationLimitExceeded
 	}
 	sc.ActiveReplicationsCounter.activeReplicatorCount++
+	db.DbStats.Database().NumConcurrentReplications.Add(1)
 	base.TracefCtx(ctx, base.KeyHTTP, "Acquired replication slot (active: %d/%d)", sc.ActiveReplicationsCounter.activeReplicatorCount, capacity)
 
 	return true, nil
 }
 
 // decrementConcurrentReplications decrements the number of active replications on the server context
-func (sc *ServerContext) decrementConcurrentReplications(ctx context.Context) {
+func (sc *ServerContext) decrementConcurrentReplications(ctx context.Context, db *db.Database) {
 	// lock replications config limit + the active replications counter
 	sc.ActiveReplicationsCounter.lock.Lock()
 	defer sc.ActiveReplicationsCounter.lock.Unlock()
 	connections := sc.ActiveReplicationsCounter.activeReplicatorLimit
 	sc.ActiveReplicationsCounter.activeReplicatorCount--
+	db.DbStats.Database().NumConcurrentReplications.Add(-1)
```

Comment:
move outside the lock, since this is an atomic that doesn't require the `ActiveReplicationsCounter` lock.